src/status_im/utils/crypt.cljs

```diff
@@ -18,9 +18,9 @@
   (byteArrayToHex (.digest sha-256)))
 
 (defn gen-random-bytes [length cb]
-  (.randomBytes rn-dependencies/random-bytes
+  (comment((.randomBytes rn-dependencies/random-bytes
```

Comment:
Probably can be uncommented since random-bytes works now :)

Reply:
right! thanks!